apps/client/src/hoc/withAutofillData.js

```diff
@@ -15,7 +16,13 @@ const withAutofillData = (Component) =>
     const address = sessionContext[topic.slug]?.address;
 
     // Only autofill data if this topic is an STTR-flow
-    if (topic.sttrFile) {
+    if (topic.flow === Flow.olo) {
+      // olo flow
+      if (!address) {
+        console.warn("No address found, redirecting to location page");
+        return <Redirect to={geturl(routes.location, topic)} />;
```

Comment:
`routes.location`? We don't have that anymore. I think this should be `routes.intro` then? (I understand you copy/pasted, but it's good to fix it)

Reply:
Will do.